stl/src/winapisupp.cpp

```diff
@@ -548,6 +564,7 @@ extern "C" PVOID __KERNEL32Functions[eMaxKernel32Function] = {0};
 
 static int __cdecl initialize_pointers() {

```

Comment:
Pre-existing: Anyone know why this `static` function is explicitly `__cdecl`?

Reply:
It's used in `_CRTALLOC(".CRT$XIC") static _PIFV pinit = initialize_pointers;` at the bottom of the file, so external machinery is expecting a `__cdecl` function pointer.